### DISCUSSION TITLE
Transcription audio with advertisement

### DISCUSSION THREAD

**Discussion by marcomoauro** [deepgram/community#452] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

Hi all,
there is a bug in the trascription when I convert this audio https://traffic.megaphone.fm/BSMT3769932184.mp3?updated=1692696344.

it seems as if the initial part where the advertisement is. is skipped. If this was the expected behavior I understand it, the problem is that the paragraph timestamps are also shifted relative to the audio.

Thanks for the support

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

curl -X POST 'https://api.deepgram.com/v1/listen?model=nova-2&detect_language=true&smart_format=true&punctuate=true&diarize=true&paragraphs=true' -H 'Content-Type: application/json;charset=utf-8' -H 'Authorization: Token <API_TOKEN>' -d '{"url":"https://traffic.megaphone.fm/BSMT3769932184.mp3?updated=1692696344"}'

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

766f0ccd-890b-44dc-a59c-a12e785637b8

### If possible, please attach your code or paste it into the text box.

this is the response payload:

[payload.json](https://github.com/deepgram/community/files/13399520/payload.json)

### If possible, please attach an example audio file to reproduce the issue.

https://traffic.megaphone.fm/BSMT3769932184.mp3?updated=1692696344

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jjmaldonis** [comment, marked as answer]
Hey @marcomoauro, when I clicked on the link to the audio and listened to it, the first few words were the same as the transcript you posted. I don't speak Italian so it's hard for me to follow the rest.

Podcast advertisements are a tricky thing. I know a little about how they work, and my understanding is that advertisements in podcast audio get inserted dynamically. This allows podcast hosts to dynamically update the ads in their (old) podcasts with new ads. The raw audio files do not include ads, and instead the ads are inserted into the audio dynamically when the podcast audio is "downloaded".

I'm surprised the audio you linked does not include any advertisement. I have only seen the scenario where a stored transcript gets out of sync with a podcast's audio after the podcast's advertisement has been updated.

Does this situation apply to your audio file? It's difficult for me to debug without speaking Italian.
